rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -536,9 +536,12 @@ def _query(self, method: Callable, call_order: Sequence[WeightedNode], **kwargs:
             return result
 
         # no node in the call order list was succesfully queried
-        raise RemoteError(
+        log.error(
             f'Failed to query {method!s} after trying the following '
-            f'nodes: {[str(x) for x in call_order]}. Check logs for details.',
+            f'nodes: {[str(x) for x in call_order]}',
+        )
+        raise RemoteError(
+            f'All connected nodes were queried and no response was returned'
```

Comment:
Since this message is shown to the user will be find to have information about the chain name where the error happened (is in the attributes of the class) and maybe guiding the user about what to do, I believe checking the network and the active node in settings